packages/block-template/src/dev.tsx

```diff
@@ -12,7 +12,14 @@ const node = document.getElementById("app");
 
 const App = () => (
```

Comment:
It’s a little confusing to see `App` here while there exists `export const App` in `app.tsx`. How about using this PR as an opportunity to reduce ambiguity?

Reply:
https://github.com/blockprotocol/blockprotocol/pull/336/commits/8b7f464a28817647081e767105e5d98f5c7fda6f